### PR TITLE
Added the argument --quiet and -q so then when used with a regular user there is no output to the screen.

### PR DESCRIPTION
**Pull request by gdoucet**
Pull request to fix fix issue #3154.

### CODE REVIEW

**Comment by bmw** [comment]
What's the use case here? You've configured `sudo`/`su` to not prompt for a password? What's the benefit of doing this vs. configuring `cron` to run `certbot-auto` as `root`?

**Comment by gdoucet** [comment]
I know it is possible to use run certbot-auto as root, but on some system the application support team don't have access to root or the crontab from root.

**Comment by pde** [comment]
We should either merge this or [fix our renewal instructions](https://certbot.eff.org/#ubuntutrusty-apache) to specify that that has to be in root's crontab.  I think merging is preferable.

**Comment by gdoucet** [comment]
Their is really no technical reason that certbot-auto need to run as root, because certbot-auto automatically detect it is a regular user and execute the proper command with sudo.

File: certbot-auto [hunk 1]
```diff
@@ -34,7 +34,8 @@ Help for certbot itself cannot be provided until it is installed.
   -n, --non-interactive, --noninteractive   run without asking for user input
   --no-self-upgrade                         do not download updates
   --os-packages-only                        install OS dependencies and exit
-  -v, --verbose                             provide more output
+  -q, --quiet                               provide no output
```
Comment (bmw) [inline certbot-auto]: Output is still provided when `certbot-auto` upgrades itself. What if we changed the flags description to something like "provide only update/error output" (`certbot` itself will provide error output when given `-q`)?

**Comment by bmw** [comment]
After talking to pde about this out of band, I'm fine adding this. I previously implemented this exact thing in #3015 but decided against it and instead wrote up #3016.

With that said, I'm fine moving forward this PR. I made a small suggestion about the help text for the `--quiet` flag.

Additionally, while perhaps a little overly complex, these changes should actually be made to `letsencrypt-auto-source/letsencrypt-auto.template`. After making changes to this file, `build.py` should be run in the same directory which will modify `letsencrypt-auto-source/letsencrypt-auto`. We only make changes to `certbot-auto` and `letsencrypt-auto` during releases so all changes to these files should be removed.

After making these changes, remove the "[needs revision]" from the title and we'll review again. Thanks @gdoucet for the PR and holding your ground convincing us it's a good idea.

**Comment by gdoucet** [comment]
@bmw thanks for the instructions. I have change my commit and also put your usage suggestion for the --quiet argument.

**Comment by bmw** [comment]
LGTM. Thanks for the PR @gdoucet.

**Pull request merged**